app/controllers/application_controller.rb

```diff
@@ -85,7 +85,7 @@ def confirm_two_factor_authenticated
 
   def confirm_security_questions_setup
     return unless UserDecorator.new(current_user).needs_security_questions?(session)
-
+    flash.keep(:notice)
```

Comment:
It was necessary for specs to pass; the previous flash was dropped when we added the error to update security questions.